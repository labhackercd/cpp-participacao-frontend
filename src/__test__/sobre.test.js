import React from 'react';
import { mount } from 'enzyme';
import MockTheme from './mocks/mockTheme';
import Sobre from '../pages/sobre';

test('Snapshot Sobre page should not have changes', () => {
  const component = mount(<MockTheme><Sobre /></MockTheme>);
  expect(component.exists()).toEqual(true);
  expect(component).toMatchSnapshot();
});
