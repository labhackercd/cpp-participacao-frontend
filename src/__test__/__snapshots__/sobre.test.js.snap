// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Sobre page should not have changes 1`] = `ReactWrapper {}`;
